### PR TITLE
Charlotte Johansson - portfolio

### PR DESCRIPTION
**Pull request by charlottejohansson**
**Brief**

Chreating my own portfolio. This is not the most advanced project from a technical standpoint (no javascript involved) but is always one of the hardest projects to finish – because the portfolio will never be finished. The most challenging part of this project is to make sure it works in all the most commonly used web browsers but also for people who experience disabilities.

**What you need to do**

✓ The page should be built with HTML and CSS (JS if needed)

✓ Use CSS Grid or Flexbox for layout

✓ The page should be fully responsive (mobile, tablet and desktop)

✓ The page should look and function the same on Chrome, Firefox, and Edge.

✓ The page should work well for people that use screen readers and keyboard navigation.

✓ The HTML code should validate in a validation tool.

**View it live**
https://charlottejohansson-portfolio.netlify.app/**Brief**

### CODE REVIEW

**Comment by charlottejohansson** [comment]
**Brief**

Chreating my own portfolio. This is not the most advanced project from a technical standpoint (no javascript involved) but is always one of the hardest projects to finish – because the portfolio will never be finished. The most challenging part of this project is to make sure it works in all the most commonly used web browsers but also for people who experience disabilities.

**What you need to do**

✓ The page should be built with HTML and CSS (JS if needed)

✓ Use CSS Grid or Flexbox for layout

✓ The page should be fully responsive (mobile, tablet and desktop)

✓ The page should look and function the same on Chrome, Firefox, and Edge.

✓ The page should work well for people that use screen readers and keyboard navigation.

✓ The HTML code should validate in a validation tool.

**View it live**
https://charlottejohansson-portfolio.netlify.app/

File: index.html [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+        </div>
+
+            <div class="feature-projects-container">
+                <div class="feature-project-wrapper">
+                    
+                    <!--First project card-->
+                    <div class="feature-project-card">
+                        <article class="feature-card">
+                            <div class="project-image-container">
+                                <div class="overlay"></div>
```
Comment (Kristin-Larsson) [inline index.html:73]: Curious why you need this div when it's empty.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+                <p>Slack</p>
+                <p>VS Code</p>
+                <p>Notion</p>
+            </div>
+        </div>      
+    </section>
+
+    <!--More section-->
+    <section class="portfolio-section">
+        <div class="heading-container">
```
Comment (Kristin-Larsson) [inline index.html:211]: Nice to reuse same section & div class in many places.

**Review by Kristin-Larsson — APPROVED** [review]
Great job! Your portfolio looks very nice, is responsive & works well in different browsers. Your code is easy to follow & is semantic.  You have a few errors regarding link buttons in validator.w3.org otherwise it works very well.

**Pull request closed without merging**